css/style.css

```diff
@@ -0,0 +1,81 @@
+html,
+body {
+  height: 100%;
+}
+
+body {
+  font-family: Helvetica, Helvetica-Neue, Arial;
+  text-align: center;
+}
+
+.container {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
As a mixin, that would be fine. With regards to IE, you need it for IE9 only. >=IE 10 has this property as deprecated (listed to be removed)